ui/src/app/streams/stream-deploy/builder/errors/errors.component.ts

```diff
@@ -0,0 +1,44 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, OnInit, Output, } from '@angular/core';
+
+/**
+ * StreamDeployBuilderErrorsComponent
+ *
+ * @author Damien Vitrac
+ */
+@Component({
+  selector: 'app-stream-deploy-builder-errors',
+  templateUrl: 'errors.component.html',
+  styleUrls: ['styles.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class StreamDeployBuilderErrorsComponent implements OnInit {
+
+  @Input() errors: { global: Array<string>, app: Array<string> };
```

Comment:
No a super-fan of this "implied" interfaces. Just wonder if maybe creating explicit interfaces would be a bit more expressive. Maybe I have just done Java for too long :-)